src/elements/Header/Header.js

```diff
@@ -52,10 +52,13 @@ function Header(props) {
     return <ElementType {...rest} className={classes}>{children}</ElementType>
   }
 
-  if ((image && typeof image !== 'boolean') || (icon && typeof icon !== 'boolean')) {
+  const iconNode = Icon.create(icon)
+  const imageNode = Image.create(image)
```

Comment:
I've been moving these names to `fooElement`.  We had a number of `fooJSX` previously.  I think element is most accurate since factories return a ReactElement.  LMK if you disagree on the name, but I do want to make them consistent.